src/Serilog.Sinks.RollingFile/Sinks/RollingFile/Specifier.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+
+namespace Serilog.Sinks.RollingFile.Sinks.RollingFile
+{
+    internal class Specifier
+    {
+        internal const string OldStyleDateToken = "{0}";
+
+        internal const string DateToken = "{Date}";
```

Comment:
It will be tidier to expose the specifiers instead, so these can just be `Specifier.Date.Token`.